packages/run-protocol/test/amm/vpool-xyk-amm/test-xyk-amm-swap.js

```diff
@@ -453,6 +460,10 @@ test('amm doubleSwap', async t => {
     moolaR.issuer,
     'Moola',
   );
+
+  const state1 = await E(metricsNotifier).getUpdateSince(state0.updateCount);
+  t.deepEqual(state1.value, { XYK: [moolaR.brand] });
```

Comment:
and this would be,
```suggestion
  m.assertChange({ XYK: {"1": moolaR.brand } });
```
"1" to indicate that only the 1-index of the array changed

Reply:
done